backend/launch.ts

```diff
@@ -0,0 +1,316 @@
+import { PrismaClient } from '@prisma/client'
+
+const db = new PrismaClient()
+
+main().catch((e) => {
+  throw e
+})
+
+async function main() {
+  // Create base set of languages
+
+  await db.language.create({
+    data: {
+      name: 'Mandarin Chinese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Spanish',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'English',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Hindi',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Bengali',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Portuguese',
+      dialect: 'European',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Portuguese',
+      dialect: 'Brazilian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Russian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Japanese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Western Punjabi',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Marathi',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Telugu',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Turkish',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Korean',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'French',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'German',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Vietnamese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Tamil',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Urdu',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Javanese',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Italian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Arabic',
+      dialect: 'Egyptian',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Gujarati',
+    },
+  })
+  await db.language.create({
+    data: {
+      name: 'Farsi',
```

Comment:
This one brings up a good question of variations. Farsi is also widely known as Persian, although Farsi is the official name of the language today. It could be helpful to have a field for "variations"? 🧐